uat/src/main/resources/greengrass/features/log-manager-1.feature

```diff
@@ -40,6 +40,45 @@ Feature: Greengrass V2 LogManager
         """
         And I deploy the Greengrass deployment configuration
         Then the Greengrass deployment is COMPLETED on the device after 2 minutes
+        And I verify the aws.greengrass.LogManager component is RUNNING using the greengrass-cli
+        And I verify that it created a log group for component type GreengrassSystemComponent for component System, with streams within 120 seconds in CloudWatch
+        And I verify that it created a log group for component type UserComponent for component UserComponentA, with streams within 120 seconds in CloudWatch
+
+    @R1    @functional @M2 @B1 @stable
+    Scenario: LogManager-1-T3: As a customer I can configure the logs uploader to delete log oldest log files inorder to keep the disk space limit configured by the customer
+        Given I create a Greengrass deployment with components
+            | aws.greengrass.LogManager | LATEST
```

Comment:
This is wrong you are missing a closing `|` make sure you run this test before pushing